requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+-r app/backend/requirements.txt
```

Comment:
When should requirements-dev.txt be run? Should we update instructions/scripts?

Reply:
Only if you're running the tests. As mentioned in the PR, I didn't add a workflow file in this PR since there's one in Anthony's PR, but I'm also happy to do it in this PR as well. Right now these tests are opt-in.

I can also add a README about running tests, but it doesn't feel that useful for the average developer until we have much higher test coverage.